rbac/providers/ldap/delta_inbound_sync.py

```diff
@@ -86,91 +79,56 @@ def fetch_ldap_data(data_type):
         search_filter=search_filter,
         attributes=ldap3.ALL_ATTRIBUTES,
     )
-    for entry in ldap_conn.entries:
-        LOGGER.info(entry.whenChanged.value)
 
-    insert_to_db(data_dict=ldap_conn.entries, data_type=data_type)
-    provider_id = LDAP_DC
-    # TODO: Save one time for delta sync
-    save_sync_time(provider_id, sync_source, "delta")
+    parsed_last_sync_time = datetime.strptime(last_sync_time.split("+")[0], "%Y-%m-%dT%H:%M:%S.%f").replace(tzinfo=timezone.utc)
+    insert_to_db(data_dict=ldap_conn.entries, when_changed=parsed_last_sync_time) 
 
 
-def insert_to_db(data_dict, data_type):
+def insert_to_db(data_dict, when_changed):
     """Insert (Users | Groups) individually to RethinkDB from dict of data and begins delta sync timer."""
+    insertion_counter = 0
     for entry in data_dict:
-        entry_data = json.loads(entry.entry_to_json())["attributes"]
-        if data_type == "user":
-            standardized_entry = inbound_user_filter(entry_data, "ldap")
-        elif data_type == "group":
-            standardized_entry = inbound_group_filter(entry_data, "ldap")
-        inbound_entry = {
-            "data": standardized_entry,
-            "data_type": data_type,
-            "timestamp": datetime.now().replace(tzinfo=timezone.utc).isoformat(),
-            "provider_id": LDAP_DC,
-        }
-        r.table("queue_inbound").insert(inbound_entry).run()
```

Comment:
`rbac-providers-ldap    | rethinkdb.errors.ReqlOpFailedError: Table `rbac.queue_inbound` does not exist in:`

I think the table name needs to be changed

Reply:
Nadia is working on a PR with a bugfix, so I'm approving.